brenda-johnson.html

```diff
@@ -0,0 +1,45 @@
+<html>
+<head>
+    <title>CSS Timeline</title>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="brenda-johnson.css">
+</head>
+
+<body>
+    <h1>Brief History of CSS</h1>
+    <ul>
+        <li id="green"><time class="time">1994</time>: The first work on style sheets started at CERN.</li>
```

Comment:
Hmm, I don't think the `<time>` element is appropriate here according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).